includes/post-teaser.html

```diff
@@ -0,0 +1,30 @@
+{%- assign postTeaserLang = page.lang | default: site.lang | default: "de" -%}
+{%- assign postTeaserPost = include.post -%}
+{%- assign postTeaserTag = include.tag -%}
+{%- assign excerpt = post.excerpt | default: post.content -%}
+
+{%- capture postTeaserPostString -%}
```

Comment:
Könnte man hier die Logik in Vue umsetzen? Selbes Ding mit dem Default im Excerpt

Reply:
Könnte man, aber habe hier explizit darauf gesetzt, weniger Daten an die Komponente zu übergeben, weil die Post-Teaser in einer großen Menge auf der Seite sind und ich so wenig Daten wie möglich rüber geben wollte. Baue ja auch das JSON Objekt, deshalb extra im Liquid Template zusammen.